webapp/src/components/modals/styles.css

```diff
@@ -0,0 +1,284 @@
+.modal {
+  opacity: 1;
+  padding-top: 12%;
+}
+
+.modal-open .modal {
+  overflow-x: hidden;
+  overflow-y: hidden;
+}
+
+.modal-dialog .modal-header {
+  display: flex;
+}
+
+.modal-dialog .modal-body {
+  display: flex;
+  flex-direction: column;
+}
+
+.modal-dialog {
+  width: 615px;
+  max-width: 615px;
+}
+
+.customModalBody {
+  height: 80%;
+  overflow-y: auto;
+}
+
+@keyframes fade-in {
+  from {
+    opacity: 0;
+  }
+  to {
+    opacity: 1;
+  }
+}
+
+@keyframes fade-out {
+  from {
+    opacity: 1;
+  }
+  to {
+    opacity: 0;
+  }
+}
+
+.fade-enter {
+  opacity: 0;
+  animation: fade-in 300ms forwards;
+}
+
+.fade-exit {
+  opacity: 1;
+  animation: fade-out 300ms forwards;
+}
+ 
+.btn-group {
+  display: flex;
+  justify-content: space-between;
+}
+
+.btn-group .btn.btn-default {
+  background: none;
+  border-color: transparent;
+  border: transparent;
+}
+
+.btn-group .btn.btn-default:hover, .btn-group .btn.btn-default:focus {
+  background: transparent;
+  color: #000000;
+}
+
+.radio.btn-group {
+  justify-content: flex-start;
+}
+
+.options .btn.btn-primary {
+  background: transparent;
+}
+
+.radio {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.radio-form {
+  margin-bottom: 0px;
+}
+
+.add-actions {
+  margin-top: 15px;
+}
+
+.validation-warning {
+  color: red;
+  font-size: 12px;
+}
+
+.config-form {
+  width: 100%;
+}
+
+.add-action-button {
+  display: block;
+}
+
+.guestButton.active {
+  box-shadow: none;
```

Comment:
Isn't this by default?

Reply:
no, radio buttons have a shadow by default